### PR TITLE
COST-1986: Remove deprecated project from gcp endpoints.

### PR DESCRIPTION
**Pull request by myersCody**
Clean up from the naming collision problem we had while working on OCP on GCP
- https://issues.redhat.com/browse/COST-1986

### CODE REVIEW

File: koku/api/tags/gcp/queries.py [hunk 1]
```diff
@@ -34,8 +34,8 @@ class GCPTagQueryHandler(TagQueryHandler):
     ]
     TAGS_VALUES_SOURCE = [{"db_table": GCPTagsValues, "fields": ["key"]}]
     # TODO: COST-1986
```
Comment (maskarb) [inline koku/api/tags/gcp/queries.py:36]: Can we remove this comment?

**Review by maskarb — CHANGES_REQUESTED** [review]
One nit-pick:

**Review by lcouzens — APPROVED** [review]
LGTM, the last smoke run had 4 failures but speaking to QE they are expected on main anyway.

**Pull request merged**